shell/platform/fuchsia/flutter/flatland_external_view_embedder.cc

```diff
@@ -235,8 +235,29 @@ void FlatlandExternalViewEmbedder::SubmitFrame(
         const ViewMutators view_mutators =
             ParseMutatorStack(view_params.mutatorsStack());
         const SkSize view_size = view_params.sizePoints();
-        FML_CHECK(view_mutators.total_transform ==
-                  view_params.transformMatrix());
+
+        // Verify that we're unpacking the mutators' transform matrix correctly
+        // on debug builds. Use built-in get method for SkMatrix to get values
+        // See:
+        // https://source.corp.google.com/piper///depot/google3/third_party/skia/HEAD/include/core/SkMatrix.h;l=391
+#ifdef NDEBUG
```

Comment:
Doing this on only debug builds is useless because we never run C++ debug builds, even during developement.

Please remove the #ifdef entirely and change the FML_LOG below from FATAL to an ERROR.

Reply:
Done